src/api/routes/ready.ts

```diff
@@ -1,7 +1,8 @@
 import type express from 'express';
 
-import { isReady } from 'lib/helpers/isReady';
+import { tasksManager } from 'lib/tasksManagerSingleton';
 
 export function ready(req: express.Request, res: express.Response): void {
```

Comment:
actually it was calling isReady in both function, not sure why it was different.
Kubernetes allows you to make a difference but we usually don't use that, it could be one single endpoint

Reply:
Ah ok i didn't know that was for kube